File: haushaltsbuch-abfrage/src/test/java/com/github/haschi/haushaltsbuch/abfrage/RunCukesTest.java
package com.github.haschi.haushaltsbuch.abfrage;

import cucumber.api.CucumberOptions;
import cucumber.api.junit.Cucumber;
import org.junit.runner.RunWith;

@RunWith(Cucumber.class)
@CucumberOptions(tags = {"@domäne", "~@ignore"}, strict = true)
public class RunCukesTest
{
}

File: infrastruktur/src/test/kotlin/com/github/haschi/domain/haushaltsbuch/RunCukesTest.kt
package com.github.haschi.domain.haushaltsbuch

import cucumber.api.CucumberOptions
import cucumber.api.junit.Cucumber
import cucumber.deps.com.thoughtworks.xstream.annotations.XStreamConverter
import com.github.haschi.domain.haushaltsbuch.testing.MoneyConverter
import org.junit.runner.RunWith

@XStreamConverter(MoneyConverter::class)
@CucumberOptions(
        tags = ["~@ignore"],
        plugin = ["pretty"])
@RunWith(Cucumber::class)
class RunCukesTest

